tests/testthat/test-git.R

```diff
@@ -0,0 +1,22 @@
+test_that("Calling git_checkout with correct branch name and an exist local path of github project returns list of string", {
+  git_repo_path <- suppressWarnings(git_create_sample_log())
```

Comment:
@nicoelee123 Could you confirm why suppressWarnings is needed here?

Also, this may be the function you need, instead of tryCatch, to handle the warning being generated on the problematic unit test in Leilani's PR.

Reply:
@carlosparadis The suppressWarnings are there becuase the git_create_sample_log() and git_delete_sample_log() throws a warning everytime and I'm not sure why. I assumed it would be okay to suppress because despite the warnings, the sample log is successfully created and deleted. 

![image](https://user-images.githubusercontent.com/63322470/235331127-c7f812c2-9ff6-4a07-985d-4201d7cfd327.png)